.github/workflows/run-tools-onnx-subgr-build.yml

```diff
@@ -0,0 +1,64 @@
+name: Run tools/onnx-subgraph Ubuntu Build
+
+on:
+  push:
+    branches:
+      - master
+      - release/*
+    paths:
+      - '.github/workflows/run-tools-onnx-subgr-build.yml'
+      - 'tools/onnx_subgraph/**'
+      - '!**/*.md'
+  pull_request:
+    branches:
+      - master
+      - release/*
+    paths:
+      - '.github/workflows/run-tools-onnx-subgr-build.yml'
+      - 'tools/onnx_subgraph/**'
+      - '!**/*.md'
+
+defaults:
+  run:
+    shell: bash
+
+# Cancel previous running jobs when pull request is updated
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  circle-mlir-test:
+    if: github.repository_owner == 'Samsung'
```

Comment:
Is this line necessary? What if this line doesn't exist?

Reply:
> What if this line doesn't exist?

clones will also run this Workflow that may not be intended.